cmd/kube-proxy/app/server.go

```diff
@@ -151,3 +176,7 @@ func (s *ProxyServer) Run(_ []string) error {
 	proxier.SyncLoop()
 	return nil
 }
+
+func (s *ProxyServer) BirthCry() {
```

Comment:
Will this ever be called in any other control path then the one on line 135?  if not then collapse.